src/ToDo/Business/IUserProfilePictureService.cs

```diff
@@ -3,5 +3,5 @@ namespace ToDo.Business;
 
 public interface IUserProfilePictureService
 {
-	Task<byte[]> GetAsync(CancellationToken ct);
+	ValueTask<byte[]> GetAsync(string userEmailAddress, CancellationToken cancellationToken);
```

Comment:
It's unclear for me why the email would be required there...

Reply:
We check to make sure that the profile picture is for this user. I guess we could have just passed in the usercontext object instead